diff -uprN xfsprogs-3.1.1.orig/libxfs/Makefile xfsprogs-3.1.1/libxfs/Makefile
--- xfsprogs-3.1.1.orig/libxfs/Makefile	2016-01-23 05:25:18.075308041 +0000
+++ xfsprogs-3.1.1/libxfs/Makefile	2016-01-23 05:28:35.347885934 +0000
@@ -41,8 +41,10 @@ default: ltdepend $(LTLIBRARY)
 include $(BUILDRULES)
 
 install: default
+	$(INSTALL_LTLIB)
 
 install-dev: default
+	$(INSTALL_LTLIB_DEV)
 
 install-qa: default
 
